views/audit/big_query_elt_script_logging.sql

```diff
@@ -0,0 +1,525 @@
+CREATE OR REPLACE VIEW `project_id.dataset_id.view_name` AS (
+WITH jobChangeEvent AS (
+  SELECT
+    protopayload_auditlog.authenticationInfo.principalEmail,
+    resource.labels.project_id AS projectId,
+    JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson, '$.requestMetadata.callerIp') AS callerIp,
```

Comment:
let's add timestamp right after callerIp since partitioning is based off timestamp.
This will let us minimize the amount of data read by view